ldm/models/diffusion/ddim.py

```diff
@@ -15,11 +15,17 @@ def __init__(self, model, schedule="linear", **kwargs):
         self.model = model
         self.ddpm_num_timesteps = model.num_timesteps
         self.schedule = schedule
+        if(torch.cuda.is_available()):
+            self.device_available = "cuda"
+        elif(torch.backends.mps.is_available()):
+            self.device_available = "mps"
```

Comment:
`torch.backends.mps` is not available in all installations of PyTorch. This needs to be either checked or surrounded with a try-catch to avoid a crash.